integration-cli/docker_cli_inspect_test.go

```diff
@@ -30,8 +31,7 @@ func (s *DockerCLIInspectSuite) OnTimeout(c *testing.T) {
 }
 
 func (s *DockerCLIInspectSuite) TestInspectImage(c *testing.T) {
-	testRequires(c, DaemonIsLinux)
-	imageTest := "emptyfs"
+	imageTest := loadSpecialImage(c, specialimage.EmptyFS)
```

Comment:
How much overlap in functionality is there between this an our "frozen images"? Is that something we should combine in the same package, or too complicated?

Which, erm, reminds me that that package/code is still somewhat broken; https://github.com/moby/moby/blob/2a38569337f97168792b8c0b5dd606032fe1dcac/testutil/fixtures/load/frozen.go#L26-L31

Reply:
I don't think we necessarily need to load these images for all the tests (like we do with the frozen images). Also, in this case a specific image content is expected. Frozen images are pulled from the registry so that's an additional layer of processing that could alter the image content/digests somehow.

But, perhaps it could be a part of the same package.